modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/js/DataSetDisplayTag.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {DataSetDisplay} from '@liferay/data-set-web';
+import React from 'react';
+
+const DataSetDisplayTag = ({
+	actionParameterName,
+	activeViewSettings,
+	apiURL,
+	appURL,
+	bulkActions,
+	creationMenu,
+	currentURL,
+	dataProviderKey,
+	filters,
+	formId,
+	id,
+	itemsActions,
+	namespace,
+	nestedItemsKey,
+	nestedItemsReferenceKey,
+	pagination,
+	portletId,
+	portletURL,
+	selectedItems,
+	selectedItemsKey,
+	selectionType,
+	showManagementBar,
+	showPagination,
+	showSearch,
+	sorting,
+	style,
+	views,
+}) => {
+	return (
+		<DataSetDisplay
```

Comment:
> We can just receive props and then pass {...props} on DataSetDisplay component.

Oh yeah, definitely :+1: 

> Also, does make sense adding PropTypes here? I think it could be useful for throwing errors in runtime(when receveing props)

No need after the `props` change :slightly_smiling_face: